### ISSUE TITLE
Tech Spec for TileDB-SOMA PyTorch support

### ISSUE THREAD

**Issue by atolopko-czi** [chanzuckerberg/cellxgene-census#427]
Write Tech Spec document to describe the proposed work needed for TileDB-SOMA PyTorch support.

**Comment by atolopko-czi** [comment]
WIP: [Tech Spec](https://docs.google.com/document/d/13mO5j33JHk8JCid_tdSLPR7np06vzqX_YlR3psxC7Zo/)

**Comment by atolopko-czi** [comment]
I've now "backfilled" a Tech Spec for the Census PyTorch DataLoader work, which was mostly done as "discovery" work and then "productionized". It may require more context/background for the uninitiated (and probably still a good bit of editing!), but I attempted to record what was ultimately built, along with some of the considerations involved. The Exploration section also contains a bunch of notes that I took as I worked through the project.

@pablo-gar @maniarathi Let me know if this is sufficient to close out this issue.
